src/HomieNode.cpp

```diff
@@ -9,8 +9,29 @@ PropertyInterface::PropertyInterface()
 : _property(nullptr) {
 }
 
-void PropertyInterface::settable(const PropertyInputHandler& inputHandler) {
+PropertyInterface& PropertyInterface::settable(const PropertyInputHandler& inputHandler) {
   _property->settable(inputHandler);
+  return *this;
+}
+
+PropertyInterface& PropertyInterface::setName(const char* name) {
```

Comment:
I'm fine with the `setXXX` syntax, that way, it stays consistent. :)

Reply:
Great Marvin!

What about the name of the function to set the retained attribute to 'false' (as it is 'true' by default)? ".retained(false)", ".non-retained()"? Maybe something non-mqtt related like ".event()"?

Reply:
Within the convention the decision was to stick to the MQTT defined wording. For consistency, just call it "setRetained()".